web/src/engine/websites/CoroCoro.ts

```diff
@@ -25,4 +63,69 @@ export default class extends DecoratableMangaScraper {
     public override get Icon() {
         return icon;
     }
+
+    public override async FetchMangas(provider: MangaPlugin): Promise<Manga[]> {
+        const scripts = await FetchCSS<HTMLScriptElement>(new Request(new URL('/rensai', this.URI)), 'script');
+        const json = this.FindAndParseJSON(scripts, 'totalChapterLikes');
```

Comment:
The `FindAndParseJSON` is always called in combination with `LocateObjectWithNamedProperty`. Consider combining both methods.

Reply:
The unique string used to find the script is not necessarily the variable property we are seeking, but i'll see what i can do.